### ISSUE TITLE
False error reported?

### ISSUE THREAD

**Issue by tomhajjar** [ra3xdh/qucs_s#415]
An error was reported during simulation but the Log only shows the term "error_amp_out" which isn't an actual error. 

Attached is the project and a ngspice test folder using DuSpice including the log file.

**TODO**

- [x] Use something like Regexp pattern for error recognition in the Ngspice log 

![2023-12-28_131642](https://github.com/ra3xdh/qucs_s/assets/102838604/e760cdcd-cb00-4896-99c2-d218d4cdc488)

[LM317_LM337_prj.zip](https://github.com/ra3xdh/qucs_s/files/13788663/LM317_LM337_prj.zip)

[LM337_ripple.zip](https://github.com/ra3xdh/qucs_s/files/13788705/LM337_ripple.zip)

**Comment by ra3xdh** [comment]
Yes, Qucs-S shows "Simulation error" message if the log contain the string "error". It is not obvious how to fix it, because Ngspice has no unified format for the error message. Maybe some RegExp would help.

**Comment by dwarning** [comment]
I am not sure and don't know the model but I think error_amp_out is a node name - the output of the regulator error amplifier. So don't worry - no ngspice problem.

**Comment by tomhajjar** [comment]
Yes the "error" is false. 

Unfortunately after I GREP'd the source I found dozens of different "Error" text strings. It would be hard to parse so many variations and the number only increases every release.

**Comment by ra3xdh** [comment]
@dwarning , The solution may be to flag "Simulation error" in the Qucs-S only if the log contains "Error" or "ERROR" strings. The "error" in lower case most probably means the node name.

**Comment by tomhajjar** [comment]
I searched the ngspice forum and GREP'd the source and found the following error strings. There are many more...

Error:
Error(
Error (
Error on line XXX:
Error on line:
Syntax error:
Expression err:
Numparam expansion errors:
Error in netlist line number...

**Comment by ra3xdh** [comment]
I have added **TODO** list for this issue.

**Comment by ra3xdh** [comment]
I have implemented pattern recognition for error messages. The node name error should not raise error flag anymore. Closing this as completed.
